ChangeLog.md

```diff
@@ -3,6 +3,308 @@
+- Added Solar Concentrators which increase laser weapon strength
+depending on local star type
+- Split up stealth part and planetary stealth techs
+- Added Flak Cannon ship part that attacks multiple times per combat
+turn but does minimal damage per attack
+- Added more AI greeting messages, made random
+- Added Scrying Sphere buildings, which can be produced and appear
+around the map randomly during galaxy generation, and share vision
+between planets on which they are located
+- The Honeycomb special will now turn nearby planets into asteroids
+instead of destroying them
+- Removed the special Volcanic Activity
+- Tweaked planet / asteroid naming
+
+
+#### Balance
+
+- Various building / tech cost adjustments
+- Split Ground Troops species trait into Defensive Ground Troops and
+Offensive Ground Troops.
+- Made Neutronium Forge only be buildable at locations with a Basic
+Shipyard (like all other shipyard upgrade buildings)
+- Tweaked monster detection and stealth stats
+- Artificial Moons can no longer be built when a Resonant Moon is present
+- Reworked various modifiers to planet population
+- Tweaked Honeycomb and World Tree special locations
+- Re-Added the Tidally Locked Rotation special with a population
+penalty and changed the penalty of the Eccentric Orbit special to a
+Supply minus
+- Space Monster weapon balance tweaks
+- Planetary Starlane Drive modified so it can't be built on gas giants
+- Xenoresurrection Lab building modified so that only one can be built
+on a planet
+- Death Spores and Bio-Terminators now only affect organic species
+- Monsters now only bombard one planet at a time
+- Adjusted Experimentor monster weapons
+- Reworked how derelict visibility effect works - grants visibility
+directly instead of temporarily increasing ship detection range
+- Added preferred focus for most species
+- Reworked supply propagation mechanics
+ - In any system, at most one empire can supply ships and exchange
+production between planets.
+ - Empires' supply ranges push against eachother in neutral systems,
+with higher range supply sources blocking lower range (where range
+decays with distance from the source)
+ - Armed ships prevent enemy supply from propagating in/out of a system
+ - In the absence of armed ships, supply won't propagate into an enemy
+system where a supply source planet is located
+- Made producing troop pods require a production location with troops,
+and made producing troop pods consume troops from the production
+location
+ - Species with no offensive Ground Troops will no longer be able to
+build ships with Ground Troop pods
+- Balanced research and production bonuses
+- Reprioritised mine effects to always occur before repair effects
+- Balance pass on Robotic line and Damage Control techs
+- Balanced BioAdaptive hull to work same as Nanorobotic
+- Tweaked numbers for Logistics Facilitator
+- Prevent Krakens Nests from spawning on Asteroid belts created by the
+Honeycomb special (and similar situations)
```

Comment:
Considering that the Honeycomb special was reworked in this version to actually create those Asteroids, does this bugfix really belong into the changelog?